### ISSUE TITLE
Inconsistencies with original paper.

### ISSUE THREAD

**Issue by GLivshits** [yangxy/GPEN#165]
Hello. There are some inconsistencies with the paper and the code:
1) I dont see any feature matching loss in your code.
2) PPL and R1 losses are not mentioned in paper, but used in your training.
3) Batch size of 1 mentioned in paper, but standard batch size choice in your code is 4.
4) It is not clear for how long to train the model.
Can you please clarify those moments? It seems like the code used in paper and the code that is allowed to show publically are quite different.

**Comment by fmu2** [comment]
A somewhat related question...

Can you please point us to the pre-trained weights that reproduce the exact model described in the paper? I am asking because we want to fairly compare with your method. It is unclear whether the provided weights are trained using private data (in addition to the publicly available FFHQ dataset) or techniques introduced after acceptance of the paper.
